### PR TITLE
update dygraph and fleetrun doc

### PR DESCRIPTION
**Pull request by danleifeng**
update dygraph and fleetrun doc

### CODE REVIEW

File: docs/source/paddle_fleet/fleetrun_usage.md [hunk 1]
```diff
@@ -59,117 +82,193 @@ fleetrun --ips="xx.xx.xx.xx,yy.yy.yy.yy" train.py
...
-```sh
- # 2个servers 8个workers
- fleetrun --servers="xx.xx.xx.xx:6170,yy.yy.yy.yy:6171" --workers="xx.xx.xx.xx:6172,xx.xx.xx.xx:6173,xx.xx.xx.xx:6174,xx.xx.xx.xx:6175,yy.yy.yy.yy:6176,yy.yy.yy.yy:6177,yy.yy.yy.yy:6178,yy.yy.yy.yy:6179" train.py
-```
+#### 在GPU集群运行参数服务器
 
-- **参数服务器训练 - 在PaddleCloud上提交任务**
+- **参数服务器训练 - 单机模拟分布式训练**
+  
+    > 1台机器通过多进程模拟，2个服务节点搭配4个训练节点， 每个训练节点占用一张GPU卡
```
Comment (wangxicoding) [inline docs/source/paddle_fleet/fleetrun_usage.md]: 可以说明一下server是否占用GPU

File: docs/source/paddle_fleet/fleetrun_usage.md [hunk 2]
```diff
@@ -59,117 +82,193 @@ fleetrun --ips="xx.xx.xx.xx,yy.yy.yy.yy" train.py
...
+    # 每台机器均指定了可用设备 GPU:0,1,2,3
+    export CUDA_VISIBLE_DEVICES=0,1,2,3
+    fleetrun --servers="xx.xx.xx.xx:6170,yy.yy.yy.yy:6171" --workers="xx.xx.xx.xx:6172,xx.xx.xx.xx:6173,xx.xx.xx.xx:6174,xx.xx.xx.xx:6175,yy.yy.yy.yy:6176,yy.yy.yy.yy:6177,yy.yy.yy.yy:6178,yy.yy.yy.yy:6179" train.py
+    ```
+
+#### 异构集群运行参数服务器
+
+- **参数服务器训练 - 单机模拟分布式训练**
+
+    > 1台机器通过多进程模拟，2个服务节点搭配2个训练节点以及2个异构训练节点，每个异构训练节点占用一张GPU卡
```
Comment (wangxicoding) [inline docs/source/paddle_fleet/fleetrun_usage.md]: 训练节点是否占用GPU

File: docs/source/paddle_fleet/fleetrun_usage.md [hunk 3]
```diff
@@ -59,117 +82,193 @@ fleetrun --ips="xx.xx.xx.xx,yy.yy.yy.yy" train.py
...
 - GPU模式相关参数:
-	- ips （str，可选）： 指定选择哪些节点IP进行训练，默认为『127.0.0.1』, 即会在本地执行单机单卡或多卡训练。
+    - ips （str，可选）： 指定选择哪些节点IP进行训练，默认为『127.0.0.1』, 即会在本地执行单机单卡或多卡训练。
 	- gpus（str, 可选）： 指定选择哪些GPU卡进行训练，默认为None，即会选择`CUDA_VISIBLE_DEVICES`所显示的所有卡。
 
 - 参数服务器模式可配参数:
-	- server_num（int，可选）：本地模拟分布式任务中，指定参数服务器服务节点的个数
-	- worker_num（int，可选）：本地模拟分布式任务中，指定参数服务器训练节点的个数
+	- server_num（int，可选）：单机模拟分布式任务中，指定参数服务器服务节点的个数
+	- worker_num（int，可选）：单机模拟分布式任务中，指定参数服务器训练节点的个数
```
Comment (wangxicoding) [inline docs/source/paddle_fleet/fleetrun_usage.md:179]: heter_worker_num 没有加

File: docs/source/paddle_fleet/fleetrun_usage.md [hunk 4]
```diff
@@ -59,117 +82,193 @@ fleetrun --ips="xx.xx.xx.xx,yy.yy.yy.yy" train.py
...
+
+    paddle.enable_static()
+    train_data, test_data, x, y, cost, acc = mnist_on_mlp_model()
+    place = paddle.CUDAPlace(int(os.environ.get('FLAGS_selected_gpus', 0)))
+    train_dataloader = paddle.io.DataLoader(
+        train_data, feed_list=[x, y], drop_last=True,
+        places=place, batch_size=64, shuffle=True)
+    fleet.init(is_collective=True)
+    strategy = fleet.DistributedStrategy()
+    #optimizer = paddle.optimizer.Adam(learning_rate=0.01)
```
Comment (wangxicoding) [inline docs/source/paddle_fleet/fleetrun_usage.md]: 这个是不是考虑干掉

**Review by gavin1332 — APPROVED** [review]
LGTM for fleet_dygraph_quick_start_cn.md

**Review by wangxicoding — APPROVED** [review]
LGTM

File: docs/source/paddle_fleet/fleetrun_usage.md (hunk 1 above)
Comment (danleifeng) [reply docs/source/paddle_fleet/fleetrun_usage.md]: done

File: docs/source/paddle_fleet/fleetrun_usage.md (hunk 2 above)
Comment (danleifeng) [reply docs/source/paddle_fleet/fleetrun_usage.md]: 不使用，done

File: docs/source/paddle_fleet/fleetrun_usage.md (hunk 3 above)
Comment (danleifeng) [reply docs/source/paddle_fleet/fleetrun_usage.md:179]: done。

File: docs/source/paddle_fleet/fleetrun_usage.md (hunk 4 above)
Comment (danleifeng) [reply docs/source/paddle_fleet/fleetrun_usage.md]: done
